src/experimental/tools/mediaCapabilitiesProber/probers/mediaDisplayInfos/index.ts

```diff
@@ -14,44 +14,37 @@
  * limitations under the License.
  */
 
-import { IMediaConfiguration } from "../../types";
-import formatConfigFor_matchMedia_API from "./format";
+import {
+  IMediaConfiguration,
+  ProberStatus,
+} from "../../types";
 
 /**
+ * @param {Object} config
  * @returns {Promise}
  */
-function isMatchMediaAPIAvailable(): Promise<void> {
-  return new Promise((resolve) => {
-    if (!("matchMedia" in window)) {
+export default function probeMatchMedia(
+  config: IMediaConfiguration
+): Promise<[ProberStatus]> {
+  return new Promise(() => {
+    if (typeof window.matchMedia !== "function") {
       throw new Error("MediaCapabilitiesProber >>> API_CALL: " +
         "matchMedia not available");
     }
-    resolve();
-  });
-}
+    if (config.display == null || !config.display.colorSpace) {
```

Comment:
I guessed that this would be an error from the previous code.

@grenault73  can you check that this is the right code?